### ISSUE TITLE
Can’t restart twice on Linux

### ISSUE THREAD

**Issue by Senyoret1** [skycoin/skywire#1640]
**Describe the bug**
If you use the option for restarting the visor and then try to us eit again, it does not work.

**Environment information:**

- OS: Linux
- Platform: Linux 4.15.0-144-generic x86_64

**Steps to Reproduce**

1. Run a visor.
2. Use the `Reboot` option on the UI (you can also use the API if you want).
3. Wait for the visor to be restarted.
4. Repeat step 2. This time it will fail with this error: `Fork/exec /bin/sh: operation not permitted`.

**Actual behavior**
The second try to restart the visor fails.

**Expected behavior**
It should be possible to restart the visor.

**Comment by 0pcom** [comment]
I'm not sure is visor restart or visor reload were ever implemented correctly.

We used to have a way that the visor would restart for updates, I think.

But all the updater code was removed, and what was left were some api endpoints that were added as cli commands just because they existed.

In the packages, currently, the process running via a systemd service, it's automatically restarted on shutdown (currently) regardless of exit status because there were some instances of exit without errors.
